src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,54 @@
+package duke.task;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+/**
+ * A type of Task, which has a deadline the task is due by.
+ */
+public class Deadline extends Task {
+
```

Comment:
Maybe you could give each of your class attributes a short description. 👍

Reply:
Thank you for your feedback! Would definitely add more descriptions in the next commits. 👍